activesupport/lib/active_support/jsonapi.rb

```diff
@@ -0,0 +1 @@
+require 'active_support/jsonapi/decoding'
```

Comment:
Encoding is a bit complicated in this context because it would need assumptions on the object (for instance it has an `id`), and the case of object attributes would be unclear (should they appear as relationships or attributes within the JSON API document?).